charmhelpers/contrib/openstack/cert_utils.py

```diff
@@ -226,8 +239,10 @@ def process_certificates(service_name, relation_id, unit,
     data = relation_get(rid=relation_id, unit=unit)
     ssl_dir = os.path.join('/etc/apache2/ssl/', service_name)
     mkdir(path=ssl_dir)
-    name = local_unit().replace('/', '_')
-    certs = data.get('{}.processed_requests'.format(name))
+    certs = data.get('{}.processed_requests'.format(get_unit_name()))
+    if not certs:
+        name = local_unit().replace('/', '_')
+        certs = data.get('{}.processed_requests'.format(name))
```

Comment:
Not sure I understand how the if not certs codepath is different to L242?

Reply:
Ah yes, sorry, I was originally including the model name in the unit name but I removed it to be consistent with the fix in the interface.